src/Settings/index.tsx

```diff
@@ -92,42 +115,27 @@ const Settings: FunctionComponent = () => {
   return (
     <ScrollView style={style.container}>
       <View style={style.section}>
-        <TouchableOpacity
+        <ListItem
```

Comment:
Should this be moved also to the external declarations like the rest of the items on the list?

Reply:
Good call, we'll also move the `ENDebugMenu` there so they're all in one place.